include/fs/littlefs.h

```diff
@@ -17,6 +17,16 @@
 extern "C" {
 #endif
 
+static inline bool littlefs_on_blkdev(struct fs_mount_t *mountp)
+{
+#ifdef CONFIG_FS_LITTLEFS_BLK_DEV
+	if (!strcmp(mountp->storage_dev, "SDMMC")) {
```

Comment:
This should be probably configurable, in case there is more than one SD mounted, or there is FAT FS mounted in parallel.

Reply:
The `"SDMMC"` was required to show this working RFC example. I can add `CONFIG_FS_LITTLEFS_BLK_DEV_MNT = 'SDMMC'` variable to make it configurable.